alacritty/src/event.rs

```diff
@@ -488,8 +488,15 @@ impl<N: Notify> Processor<N> {
                     CloseRequested => processor.ctx.terminal.exit(),
                     Resized(lsize) => {
                         let psize = lsize.to_physical(processor.ctx.size_info.dpr);
-                        processor.ctx.resize_pending.dimensions = Some(psize);
-                        processor.ctx.terminal.dirty = true;
+                        // Drop resize events, which resize to the current size
+                        if (psize.height as f32 - processor.ctx.size_info.height).abs()
```

Comment:
This change is an optimization for resizes, that are identical to a current window size. If you don't like it, I'm fine with ignoring it.

Reply:
When is this ever hit? Why would a resize event be sent in that case?

Reply:
Change focus on sway. So if you have two windows next to each other and you change a focus, you'll get a resize.

Reply:
That sounds like a winit bug to me?

Reply:
Oh, it does actually. Doesn't happen on `X11`. Should I revert this `resize` check?

Reply:
Yeah and report upstream probably. We shouldn't be checking if resize events make sense, that's something winit should be doing.

Reply:
https://github.com/rust-windowing/winit/issues/1247